src/IIIFPresentation/API.Tests/Integration/DeleteCollectionTests.cs

```diff
@@ -0,0 +1,236 @@
+﻿using System.Net;
+using Amazon.S3;
+using API.Tests.Integration.Infrastructure;
+using Core.Helpers;
+using Core.Response;
+using IIIF.Presentation.V3.Strings;
+using IIIF.Serialisation;
+using Models.API.Collection;
+using Models.API.General;
+using Models.Database.Collections;
+using Models.Database.General;
+using Models.Infrastucture;
+using Repository;
+using Test.Helpers.Helpers;
+using Test.Helpers.Integration;
+
+namespace API.Tests.Integration;
+
+[Trait("Category", "Integration")]
+[Collection(CollectionDefinitions.StorageCollection.CollectionName)]
+public class DeleteCollectionTests : IClassFixture<PresentationAppFactory<Program>>
+{
+    private readonly HttpClient httpClient;
+
+    private readonly PresentationContext dbContext;
+
+    private readonly IAmazonS3 amazonS3;
+
+    private const int Customer = 1;
+
+    public DeleteCollectionTests(StorageFixture storageFixture, PresentationAppFactory<Program> factory)
+    {
+        dbContext = storageFixture.DbFixture.DbContext;
+        amazonS3 = storageFixture.LocalStackFixture.AWSS3ClientFactory();
+
+        httpClient = factory.ConfigureBasicIntegrationTestHttpClient(storageFixture.DbFixture,
+            appFactory => appFactory.WithLocalStack(storageFixture.LocalStackFixture));
+
+        storageFixture.DbFixture.CleanUp();
+    }
+    
+    [Fact]
+    public async Task DeleteCollection_ReturnsUnauthorized_WhenCalledWithoutAuth()
+    {
+        // Arrange
+        var collectionName = nameof(DeleteCollection_ReturnsUnauthorized_WhenCalledWithoutAuth);
+
+        var requestMessage =
+            HttpRequestMessageBuilder.GetPrivateRequest(HttpMethod.Delete, $"{Customer}/collections/{collectionName}");
+
+        // Act
+        var response = await httpClient.SendAsync(requestMessage);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.Unauthorized);
+    }
+
+    [Fact]
+    public async Task DeleteCollection_ReturnsForbidden_WhenCalledWithIncorrectShowExtraHeader()
+    {
+        // Arrange
+        var collectionName = nameof(DeleteCollection_ReturnsForbidden_WhenCalledWithIncorrectShowExtraHeader);
+
+        var requestMessage = new HttpRequestMessage(HttpMethod.Delete, $"{Customer}/collections/{collectionName}");
+        requestMessage.Headers.Add("X-IIIF-CS-Show-Extras", "Incorrect");
+
+        // Act
+        var response = await httpClient.AsCustomer(1).SendAsync(requestMessage);
+
+        // Assert
+        response.StatusCode.Should().Be(HttpStatusCode.Forbidden);
+    }
+
+    [Fact]
+    public async Task DeleteCollection_DeletesCollection_WhenAllValuesProvided()
+    {
+        // Arrange
+        var initialCollection = new Collection()
```

Comment:
There's a lot of repetition in Collection tests - could we use `DatabaseTestDataPopulation.AddTestCollection()` here (may need tweaked)? It'll keep the tests more concise to show what's useful.